### DISCUSSION TITLE
Why isn't recommended the use of the CLI libraries as Dev Dependencies?

### DISCUSSION THREAD

**Discussion by vrebo** [ionic-team/capacitor#4606] [Q&A]
In the documentations the example to install the CLI dependencies of capacitor shows the typical way of installation without de --dev flag

`npm install @capacitor/android`

Is there a reason for this or can I use them as dev dependencies also?

Thanks for your help.

**Comment by jcesarmobile** [comment]
`@capacitor/cli` can be installed as a dev dependency, but it's the only one

on the docs we use `npm install @capacitor/core @capacitor/cli` to make it one line, but you can install the cli as dev dependency if you want

for other capacitor packages, like core/ios/android or our plugins, they need to be installed as regular dependencies because their content is not copied anywhere, your app uses their files from node_modules and the files are needed for your app to work

**Reply by bastiW** [reply]
I created a merge request to fix this. https://github.com/ionic-team/capacitor-site/pull/348This has two advantages:

This has two advantages:
- Decrease prod package size
- Keeps project structure clean and uses npm like it was intended to use
